src/Raven.Server/Json/WebsocketStream.cs

```diff
@@ -137,10 +147,18 @@ private void ThrowOnDisposed()
         public override void Flush()
```

Comment:
We are better off throwing not supported here.
We want to be fully async here. There is already `FlushAsync` that we can use, instead, rather than mix sync & async. That is NOT a good idea to do